### PR TITLE
feat(service): add table output

### PR DESCRIPTION
**Pull request by jbrockopp**
* Added a `table()` function to `go-vela/cli/action/service`
* Added tests for the `table()` function
* Added a `wideTable()` function to `go-vela/cli/action/service`
* Added tests for the `wideTable()` function
* Added `go-vela/cli/internal/output.Dump` as an output option
* Added `go-vela/cli/internal/output.Spew` as an output option

### CODE REVIEW

File: action/service/table_test.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+	s.SetRepoID(1)
+	s.SetNumber(1)
+	s.SetName("postgres")
+	s.SetImage("postgres:12-alpine")
+	s.SetStatus("running")
+	s.SetExitCode(0)
+	s.SetCreated(1563474076)
+	s.SetStarted(1563474078)
+	s.SetFinished(1563474079)
+	s.SetHost("example.company.com")
```
Comment (wass3r) [inline action/service/table_test.go:115]:
```suggestion
	s.SetHost("host.example.com")
```
minor nit-pick but company.com is a real website :(

File: action/service/get.go [hunk 2]
```diff
@@ -26,28 +26,39 @@ func (c *Config) Get(client *vela.Client) error {
 
 	// handle the output based off the provided configuration
 	switch c.Output {
-	case "json":
-		// output the service in JSON format
+	case output.DriverDump:
```
Comment (jbrockopp) [inline action/service/get.go:29]: Each function now supports the `dump` driver from `go-vela/cli/internal/output`

File: action/service/get.go [hunk 3]
```diff
@@ -26,28 +26,39 @@ func (c *Config) Get(client *vela.Client) error {
...
+		if err != nil {
+			return err
+		}
+	case output.DriverJSON:
+		// output the services in JSON format
 		err := output.JSON(services)
 		if err != nil {
 			return err
 		}
+	case output.DriverSpew:
```
Comment (jbrockopp) [inline action/service/get.go:41]: Each function now supports the `spew` driver from `go-vela/cli/internal/output`

File: action/service/get.go [hunk 4]
```diff
@@ -26,28 +26,39 @@ func (c *Config) Get(client *vela.Client) error {
...
-		// TODO: create output.Wide function
-		//
-		// err := output.Wide(services)
-		// if err != nil {
-		// 	return err
-		// }
-	case "yaml":
-		// output the service in YAML format
+		// output the services in wide table format
+		err := wideTable(services)
```
Comment (jbrockopp) [inline action/service/get.go:49]: Adding support for `wide` output option with a new `wideTable()` function

File: action/service/get.go [hunk 5]
```diff
@@ -26,28 +26,39 @@ func (c *Config) Get(client *vela.Client) error {
...
+		// output the services in YAML format
 		err := output.YAML(services)
 		if err != nil {
 			return err
 		}
 	default:
-		// output the service in default format
-		err := output.Default(services)
+		// output the services in table format
+		err := table(services)
```
Comment (jbrockopp) [inline action/service/get.go:61]: Updated the default output option with a new `table()` function

File: action/service/table.go [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+	"github.com/go-vela/types/library"
+
+	"github.com/dustin/go-humanize"
+	"github.com/gosuri/uitable"
+)
+
+// table is a helper function to output the
+// provided services in a table format with
+// a specific set of fields displayed.
+func table(services *[]library.Service) error {
```
Comment (jbrockopp) [inline action/service/table.go:24]: This is where we add the new `table()` function

File: action/service/table.go [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+		return err
+	}
+
+	return nil
+}
+
+// wideTable is a helper function to output the
+// provided services in a wide table format with
+// a specific set of fields displayed.
+func wideTable(services *[]library.Service) error {
```
Comment (jbrockopp) [inline action/service/table.go:58]: This is where we add the new `wideTable()` function

**Pull request merged**